### ISSUE TITLE
Improve function that finds eccentricity iteratively

### ISSUE THREAD

**Issue by flaviorangel** [ross-rotordynamics/ross#364]
So far, the function that finds the eccentricity iteratively is working based on an equation by Friswell, that finds eccentricity based on the load. A suggestion is that this function resorts on our numerical functions to find forces in x and y axis and then compares them with expected results (0 in x and load in y). The error would be the maximum between both errors.

**Comment by flaviorangel** [comment]
Based solely on the force in y axis, with a tolerance of 1e-06 and an increment of eccentricity times an increment factor of  1e-03, I was able to find a proper eccentricity after 24 iterations. I used the bearing given by the fluid_flow_short_eccentricity function in our test file for the fluid flow package. I adopted the following strategies: In case the sign of (force_y - load) changes (meaning the error function passed zero), the sign of the increment is flipped and it is reduced (divided by 10). Also, in case the error is increased instead of reduced, the sign of the increment is also flipped. For such small tolerance, it would sometimes occur that the error would increase, them the sign would be changed, them the error would increase again, in an infinite loop. Because of that, the increment is also reduced when the error increases.

Iteration 24
Eccentricity: 0.00010027583609000007
Error force y: 7.62575993462633e-07

**Comment by flaviorangel** [comment]
It was not possible to reach a result after 50 iterations with a tolerance of 1e-07 because the error would increase and pass beyond zero so many times that the increment would eventually become so small that the eccentricity would no longer change.

**Comment by flaviorangel** [comment]
Following the suggestion by professor Juliana, I used an increment factor of 1e-04 and I only added to the eccentricity, expecting the forces to stay below tolerance at some point. After 50 iterations, the force x went from -0.018764536867848847 to -0.017596539308968318, always decreasing, whereas the (force y - load) went from -0.2917483127509044 to 0.2590299343631912, always increasing, passing zero at some point.

Because I didn't know for sure how these functions behaved, I tested a greater increment factor: 1e-03. The force x still only decreases, reaching -0.004994964668643576 after 50 iterations, whereas (force y - load) never stops increasing, reaching 5.753457742790253.

They keep increasing with an increment factor of 1e-02. If we subtract the eccentricity instead of adding to it, the force x keeps increasing and (force y - load) decreasing, i.e, their derivatives don't ever change sign.

These findings suggest that both error functions are somewhat linear, meaning we can't expect both to reach zero at the same point. We must minimize one of them and tolerate the other. Since (force y - load) varies faster, we should focus on it.
